composer.json

```diff
@@ -36,5 +36,6 @@
         "branch-alias": {
             "dev-master": "1.0-dev"
         }
-    }
+    },
+    "abandoned": "league/commonmark"
```

Comment:
I don't think this actually has the effect of marking the package as abandoned. That can only be done on Packagist.

Reply:
https://getcomposer.org/doc/04-schema.md#abandoned

https://github.com/composer/composer/issues/4610

Whether or not packagist has finally automated this so it respects what's in composer.json is something that will need to be verified, for sure.

It doesn't hurt anything by adding this though.

Reply:
It does hurt to add it. It is confusing, and doesn't mark the package as abandoned.

Reply:
Again, that's a packagist UI issue that needs to be verified. It doesn't change the fact that this is supported by composer now.